enginetest/insert_queries.go

```diff
@@ -75,6 +75,18 @@ var InsertQueries = []WriteQueryTest{
 		SelectQuery:         "SELECT i FROM mytable WHERE s = 'x';",
 		ExpectedSelect:      []sql.Row{{int64(999)}},
 	},
+	{
+		WriteQuery:          "INSERT INTO mytable VALUES (999, _binary 'x');",
```

Comment:
Is it easy to write a test where the _binary value of the string differs from the golang literal? This doesn't appear to be testing much as is

Reply:
I don't think so, our charset and collation story isn't robust enough. We would first have to define a difference between a binary string and utf8mb4 string, but currently we just see them both as standard strings.